parm/ice_in_template

```diff
@@ -80,7 +80,7 @@
 &thermo_nml
     kitd              = 1
   , ktherm            = 1
-  , conduct           = 'bubbly'
+  , conduct           = 'MU71'
```

Comment:
Can I change to this namelist variable before changing to your latest CICE5?

Reply:
It does not depend on any change I'm going to be making to the code; it should be fine to change it now in your runs.